tree/tree/src/TTreeCloner.cxx

```diff
@@ -644,12 +644,13 @@ void TTreeCloner::ImportClusterRanges()
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Set the TFile cache size to be used.
+/// Set the cache size used by the matching TFile.
 /// Note that the default is to use the same size as the default TTreeCache for
 /// the input tree.
-/// \param size Size of the cache. Zero disable the use of the cache.
+/// \param size Size of the cache. Zero (or a value not matching the buffer size
```

Comment:
This change ("or a value not matching ...) is inaccurate.  What the code does is delete the existing file cache if it is not matching the request size (or disabling) so that (as pointed to by the comment at the end) the file cache can be created later with the right size (or not create if the size is 0).

Reply:
fixed